chingu-coders/Voyage2-Turtles-02
# clock settings

## Purpose
* To add settings to clocks, per the issue ticket

## Approach/Fixes
* Adds advanced functionality to clocks

## Testing
* Check out the PR
* open settings tab and confirm that you can toggle between clocks, about, and contribute
* change time format and confirm that the clock changes
* change period settings and confirm that pomodoro clock periods change
